bin/middlewares/og_meta.js

```diff
@@ -0,0 +1,91 @@
+const request = require('request')
+const acceptLanguage = require('accept-language')
+const QueryError = require('../query_error')
+
+const metas = [
+  {name : 'site_name', content : 'Qwant map'},
+  {name : 'image', content : '/images/maps_opengraph.png'},
+  {name : 'url', content : 'www.qwant.com/maps'},
+]
+
+module.exports = function(config, constants) {
+  async function getPoi(poiId, locale) {
+    let id = poiId
+    let atPos = poiId.indexOf('@')
+    if(atPos !== -1) {
+      id = poiId.slice(0, atPos)
+    }
+
+    return new Promise((resolve, reject) => {
+      let idunnUrl =`${config.services.idunn.url}/v1/pois/${id}?lang=${locale.code}`
+      request(idunnUrl, {json : true}, (error, response, body) => {
+        if(error) {
+          reject(error)
+        } else if(response.statusCode === 404) {
+          resolve(null)
+        } else {
+          resolve(body)
+        }
+      })
+    })
+  }
+
+  function commonMeta(locale, req, res) {
+    res.locals.metas = metas.map(meta => meta)
```

Comment:
maybe `ogMetas` instead of `metas` in locals ?

Reply:
refactored